x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -6873,7 +6873,6 @@
     "xpack.enterpriseSearch.workplaceSearch.overviewOnboardingUsersCard.description": "検索できるように、同僚をこの組織に招待します。",
     "xpack.enterpriseSearch.workplaceSearch.overviewOnboardingUsersCard.title": "ユーザーと招待",
     "xpack.enterpriseSearch.workplaceSearch.overviewUsersCard.title": "検索できるように、同僚を招待しました。",
-    "xpack.enterpriseSearch.workplaceSearch.productCta": "Workplace Searchの起動",
     "xpack.enterpriseSearch.workplaceSearch.productName": "Workplace Search",
```

Comment:
Oh wow, I hadn't realized we already received translations for our plugin!!!! 🎉 🎉 🎉